virttest/libvirt_remote.py

```diff
@@ -0,0 +1,46 @@
+"""
+Shared code for tests that need to control libvirt on remote
+"""
+
+import logging
+
+from avocado.core import exceptions
+
+from virttest import remote
+
+from virttest.utils_test import libvirt
+
+
+def update_remote_file(params, new_key,
```

Comment:
I assume update key with value, so the input is not just new_key, but need value as well

Reply:
updated.